test.js

```diff
@@ -112,71 +112,74 @@ test('StuckTransactionsCanceller', async t => {
       assert.deepStrictEqual(sentTransactions, [])
       assert(storage.has(tx.hash))
     })
-  })
-  await t.test('cancel old transactions', async t => {
```

Comment:
this test was misplaced hierarchically, I moved it to the right place, unfortunately that makes this diff harder to read

Reply:
The test fix was straightforward however